qemu/tests/device_bit_check.py

```diff
@@ -72,11 +73,24 @@ def run(test, params, env):
 
             logging.info("Properity bit in qtree is right for %s.", option)
             if params.get("check_in_guest", "yes") == "yes":
-                pci_info = session.cmd_output("lspci")
-                pci_n = re.findall(pci_id_pattern, pci_info)
-                if not pci_n:
-                    test.error("Can't get the pci id for device")
-                cmd = "cat /sys/bus/pci/devices/0000:%s/" % pci_n[0]
+                if params.get('machine_type').startswith("s390"):
+                    ccw_id_pattern = \
+                        ccw_id_pattern + '%s' % \
```

Comment:
Why not `ccw_id_pattern + re.findallxxxx`

Reply:
well, I think this would be better to read, modified